ex03.ex

```diff
@@ -77,7 +84,14 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([h | t], v) do
```

Comment:
## -2

You can use pattern matching to remove the conditional logic:

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~